tutorials/01_install.md

```diff
@@ -2,5 +2,73 @@
 
 Next Tutorial: \ref commandline
 
-Refer to [Ignition GUI's entry page](https://ignitionrobotics.org/libs/gui)
-for install instructions.
+[Install](#install)
+
+* [Binary Install](#binary-install)
+
+* [Source Install](#source-install)
+
+    * [Prerequisites](#prerequisites)
+
+    * [Building from Source](#building-from-source)
+
+# Install
+
+We recommend following the [Binary Install](#binary-install) instructions to get up and running as quickly and painlessly as possible.
+
+The [Source Install](#source-install) instructions should be used if you need the very latest software improvements, if you need to modify the code, or if you plan to make a contribution.
+
+## Binary Install
+
+On Ubuntu Bionic, it's possible to install Ignition GUI as follows:
+
+Add OSRF packages:
+
+    echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list
+    sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
+    sudo apt update
+
+Install Ignition GUI:
+
+    sudo apt install libignition-gui<#>-dev
+
+Be sure to replace `<#>` with a number value, such as 1 or 2, depending on
+which version you need.
+
+## Source Install
+
+There are currently two versions under active development:
+
+* **Version 1**: It makes use of `QtQuick`, which allows developing more
+  modern-looking apps with animations and a mobile/web feel. Requires Ubuntu
+  Bionic or higher.
```

Comment:
We can remove this entire section (lines 40 to 44), because there's no distinction between the versions